cfme/automate/explorer/__init__.py

```diff
@@ -28,6 +28,8 @@ class datastore(Accordion):  # noqa
         tree = ManageIQTree()
 
     configuration = Dropdown('Configuration')
+    increase_button = Text(locator='//div[contains(@class, "resize-right")]')
```

Comment:
TODO (optional)

I would like to see configuration widget moved to a `toolbar` nested view, and increase_button/decrease_button moved to a `sidebar` nested view.

Only other use of this locator is in ProvisionView, where its part of the sidebar. This would provide a more consistent view structure.  Does not need to be changed here, as it would require additional changes to anything using `AutomateExplorerView.configuration`

Reply:
**Required**

On second look, this is the `widgetastic_manageiq.Splitter` widget.  This widget should be used here, with calls to `Splitter.pull_right` during navigation. Consider using `Splitter.reset` after navigation, in CFMENavigateStep.resetter() method implementation (or just in CFMENavigateStep.step() ).

If the locator for `Splitter` isn't working here (it just uses the fa-angle-left|right classes) then we should adjust the widget accordingly.

Reply:
Hmm, nice, I wasn't aware of this `Splitter`. I pushed new changes.